client/src/components/Ticketing/ticketingmanager/dashboard.tsx

```diff
@@ -50,6 +50,17 @@ const Dashboard = () => {
 
             </div>
           </a>
+          <a onClick={() => navigate('/ticketing/purchaseticket')} className="hover:scale-105 md:h-60 md:w- transition duration-300 ease-in-out border-none  bg-gradient-to-r from-green-500 to-zinc-500 rounded-3xl  shadow-xl  ">
+            <div className="m-8 flex flex-col items-start">
+              <svg xmlns="http://www.w3.org/2000/svg" fill="white" className="bi bi-credit-card h-12 w-12" viewBox="0 0 20 20">
+                <path d="M0 4a2 2 0 0 1 2-2h12a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2H2a2 2 0 0 1-2-2V4zm2-1a1 1 0 0 0-1 1v1h14V4a1 1 0 0 0-1-1H2zm13 4H1v5a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1V7z"/>
+                <path d="M2 10a1 1 0 0 1 1-1h1a1 1 0 0 1 1 1v1a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1v-1z"/>
+              </svg>
+
+              <h5 className="mb-2 ml-2 text-2xl font-bold tracking-tight text-white">Purchase Tickets</h5>
```

Comment:
You'll want to change `ml-2` to `ml-1` so that the text properly aligns with the icon.